client/src/charts/wrapped_nivo/nivo_common.js

```diff
@@ -88,15 +100,21 @@ const DefaultPercentTooltip = ({tooltip_items, formatter, total}) => (
       ({tooltip_item}) => (
         <Fragment>
           <MediaQuery minDeviceWidth={breakpoints.minSmallDevice}>
-            <td className="nivo-tooltip__content">{tooltip_item.name || tooltip_item.id}</td>
-            <td className="nivo-tooltip__content" dangerouslySetInnerHTML = {{__html: formatter(tooltip_item.value)}}/>
-            <td className="nivo-tooltip__content" dangerouslySetInnerHTML = {{__html: formats.percentage1(Math.abs(tooltip_item.value)/total)}}/>
+            <Fragment>{ /* MediaQuery jank, it will insert a div wrapping its children when it has mutliple of them, need a manual Fragment to avoid that */ }
+              <td className="nivo-tooltip__content">
+                {tooltip_item.name || tooltip_item.id}
+              </td>
+              <td className="nivo-tooltip__content" dangerouslySetInnerHTML={{__html: formatter(tooltip_item.value)}} />
+              <td className="nivo-tooltip__content" dangerouslySetInnerHTML={{__html: formats.percentage1(Math.abs(tooltip_item.value)/total)}} />
+            </Fragment>
           </MediaQuery>
           <MediaQuery maxDeviceWidth={breakpoints.maxSmallDevice}>
             <td>
-              <div className="nivo-tooltip__content">{tooltip_item.name || tooltip_item.id}</div>
-              <div className="nivo-tooltip__content" dangerouslySetInnerHTML = {{__html: formatter(tooltip_item.value)}}/>
-              <div className="nivo-tooltip__content" dangerouslySetInnerHTML = {{__html: formats.percentage1(Math.abs(tooltip_item.value)/total)}}/>
+              <div className="nivo-tooltip__content">
+                {tooltip_item.name || tooltip_item.id}
+              </div>
+              <div className="nivo-tooltip__content" dangerouslySetInnerHTML={{__html: formatter(tooltip_item.value)}} />
+              <div className="nivo-tooltip__content" dangerouslySetInnerHTML={{__html: formats.percentage1(Math.abs(tooltip_item.value)/total)}} />
```

Comment:
Yeah, hmm, I'll need to double check if either of these are necessary. Some of the formatters do return HTML strings, but they usually have a "raw" option to avoid that (might just be able to use raw here and skip the headache, if that's the case).

Reply:
Hmm, `DefaultPercentTooltip` and `DefaultTooltip` does seem pretty redundant, except this line (108 and 117). I wonder if you could add `children` (or config) props to `DefaultTooltip` and basically pass in line 117? And you could still have it as separate variable like so.

```
const const DefaultPercentTooltip = ({tooltip_items, formatter, total}) => <DefaultTooltip (...) children={
  ({tooltip_item}) => <div (...) dangerouslySetInnerHTML={{__html: 
    formats.percentage1(Math.abs(tooltip_item.value)/total)}} />
}
```
I didn't actually think this through though so not sure if it'll work. But something along this line is worth a try?